src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -232,6 +232,10 @@ public function load(array $configs, ContainerBuilder $container)
         }
 
         if ($this->isConfigEnabled($container, $config['form'])) {
+            if (!class_exists('Symfony\Component\Form\Form')) {
+                throw new LogicException('Form support cannot be enabled. Please install the Form component.');
```

Comment:
just to match similar messages: 
`Form support cannot be enabled as the Form component is not installed.`

Reply:
✅